gatsby-node.js

```diff
@@ -62,75 +64,146 @@ exports.createPages = async (createPagesArgs) => {
-    );
-    if (!description || !description.value) {
-      console.warn(
-        `No description for job "${position.name}" (${position.id}) found!`,
-      );
-    }
+        // we need to normalize the jobDescription to an array, otherwise gatsby
+        // complains about different types for the same variable
+        position.jobDescriptions.jobDescription = Array.isArray(
+          position.jobDescriptions.jobDescription,
+        )
+          ? position.jobDescriptions.jobDescription
+          : [position.jobDescriptions.jobDescription];
+
+        position.jobDescriptions.jobDescription =
+          position.jobDescriptions.jobDescription
+            .map((description) => {
+              if (description.name.trim() === 'Slug') {
+                position.satellytesPath = `/career/${description.value.trim()}/`;
+                return undefined;
+              } else {
+                return {
+                  name: description.name.trim(),
+                  value: description.value.trim(),
+                };
+              }
+            })
+            .filter((position) => position);
+
+        const description = position.jobDescriptions.jobDescription.find(
+          (description) => description.name === PERSONIO_SHORT_DESCRIPTION_NAME,
+        );
+        if (!description || !description.value) {
+          console.warn(
+            `No description for job "${position.name}" (${position.id}) found!`,
+          );
+        }
 
-    position.satellytesShortDescription = description
-      ? description.value || ''
-      : '';
-
-    /**
-     * Ideally we create first the nodes from the API and then create the pages
-     * That way we could incorporate the onCreateNode method to generate the image
-     * for career positions too. Until then we run it manually here and pass in the
-     * path to the image through the context
-     */
-    const fileName = `social-card---career-${slugify(position.name, {
-      lower: true,
-    })}.jpg`;
-    const outputFile = path.join('public', fileName);
-    const publicUrl = `${siteMetadata.siteUrl}/${fileName}`;
+        position.satellytesShortDescription = description
+          ? description.value || ''
+          : '';
+
+        /**
+         * Ideally we create first the nodes from the API and then create the pages
+         * That way we could incorporate the onCreateNode method to generate the image
+         * for career positions too. Until then we run it manually here and pass in the
+         * path to the image through the context
+         */
+        const fileName = `social-card---career-${slugify(position.name, {
+          lower: true,
+        })}.jpg`;
+        const outputFile = path.join('public', fileName);
+        const publicUrl = `${siteMetadata.siteUrl}/${fileName}`;
+
+        generateCard({ title: position.name }, outputFile);
+
+        const getTranslations = (id) => {
```

Comment:
I think we can savely rename this to `getTranslation` and only return one object, as we will not add any other language then german and english.